src/common/merge.ts

```diff
@@ -20,6 +25,54 @@ export interface PullRequestDetails {
 const EXPONENTIAL_BACKOFF = 2;
 const MINIMUM_WAIT_TIME = 1000;
 
+const getIsModified = async (
+  octokit: ReturnType<typeof getOctokit>,
+  query: {
+    pullRequestNumber: number;
+    repositoryName: string;
+    repositoryOwner: string;
+  },
+): Promise<boolean> => {
+  const iterator = makeGraphqlIterator<
+    FindPullRequestCommitsResponse,
+    PullRequestCommitNode
+  >(
+    octokit,
+    findPullRequestCommits,
+    query,
+    (
+      response: FindPullRequestCommitsResponse,
+    ): IterableList<PullRequestCommitNode> =>
+      response.repository.pullRequest.commits,
+  );
+
+  // eslint-disable-next-line immutable/no-let
+  let originalAuthor: string | undefined = undefined;
```

Comment:
Seems like we can use the `next` method for direct assignment here, avoiding the need to `let`?